samtools/1.10/Dockerfile

```diff
@@ -0,0 +1,52 @@
+  
+################## BASE IMAGE ######################
+
+FROM biocontainers/biocontainers:v1.1.0_cv2
+
+################## METADATA ######################
+
+ARG software_version=1.7.0
+LABEL base_image="biocontainers:v1.1.0_cv2"
+LABEL software="samtools"
+LABEL version="1.10"
+LABEL software.version="$software_version"
+LABEL about.summary="Tools for manipulating next-generation sequencing data"
+LABEL about.home="https://github.com/samtools/samtools"
+LABEL about.documentation="https://github.com/samtools/samtools"
+LABEL about.license_file="https://github.com/samtools/samtools"
+LABEL about.license_file="https://github.com/samtools/samtools/blob/develop/LICENSE"
+LABEL about.license="SPDX:MIT"
+LABEL about.tags="Genomics"
+LABEL extra.identifiers.biotools="SAM"
```

Comment:
is bio.tools not *samtools* (https://bio.tools/samtools) ?

Reply:
The link shows the tool is samtools, but it's not the home page of samtools. I am not sure what do you need here. I saw you used both `LABEL extra.identifiers.biotools="SAM"`  and `LABEL extra.binary = "SAMTOOLS"` before.